cerbos/model.go

```diff
@@ -750,6 +766,23 @@ func (rp *ResourcePolicy) AddResourceRules(rules ...*ResourceRule) *ResourcePoli
 	return rp
 }
 
+// WithConstantsImports adds import statements for exported constants.
+func (rp *ResourcePolicy) WithConstantsImports(name ...string) *ResourcePolicy {
+	rp.Obj.Constants.Import = append(rp.Obj.Constants.Import, name...)
+	return rp
+}
+
+// WithConstant adds a constant definition for use in conditions.
+// Valid value types are those that can be converted to a [google.protobuf.Value].
+//
+// [google.protobuf.Value]: https://pkg.go.dev/google.golang.org/protobuf/types/known/structpb#NewValue
+func (rp *ResourcePolicy) WithConstant(name string, value any) *ResourcePolicy {
+	var err error
+	rp.Obj.Constants.Local[name], err = structpb.NewValue(value)
```

Comment:
We have the `internal.ToStructPB` function that's a bit more forgiving when handling slices and maps.

Reply:
Nice, thanks!